homeassistant/components/asuswrt/config_flow.py

```diff
@@ -191,6 +199,24 @@ async def async_step_init(self, user_input=None):
 
         data_schema = vol.Schema(
             {
+                vol.Optional(
+                    CONF_CONSIDER_HOME,
+                    default=self.config_entry.options.get(
+                        CONF_CONSIDER_HOME, DEFAULT_CONSIDER_HOME.total_seconds()
+                    ),
+                ): vol.All(vol.Coerce(int), vol.Clamp(min=0, max=900)),
+                vol.Optional(
+                    CONF_TRACK_NEW,
```

Comment:
Can we replace this option with the system level option?

https://developers.home-assistant.io/docs/entity_registry_disabled_by#config-entry-system-options-setting-default-value-of-disabled_by-for-new-entity-registry-entries

Reply:
You are suggesting to remove this and use system option or something different?
I think it could work, just it is not very intuitive for the users.

Reply:
Yes, it's what we've built the system level option for.

Reply:
Test preparation is failing, is because this is a old branch?

Reply:
I can rebase the target branch and then you can rebase this PR.

Reply:
I've rebased the target branch `home-assistant:asuswrt-config-entries` so you can rebase this PR on top of that branch.